# alt-require

`alt` attribute must be present in `<img />` tags.

**good**

```html
<img src="logo.png" alt="Brand Name">
```

```html
<img src="logo.png" alt="">
```

**bad**

```html
<img src="logo.png">
```
