# attr-no-duplication

Attributes should not be duplicated.

**good**

```html
<img src="logo.png" alt="Brand Name">
```

**bad**

```html
<img src="logo.png" src="new-logo.png" alt="Brand Name">
```
