# class-ad-disabled

Class names should not contain `ad(s)` or `sponsor(s)` to avoid getting flagged by ad blockers.

**good**

```html
<p class="eventsponsor">
```

**bad**

```html
<p class="event-sponsor">
```

```html
<p class="sponsor">
```

```html
<p class="sponsors">
```

```html
<p class="ad">
```

```html
<p class="ads">
```
