# id-ad-disabled

IDs should not contain `ad(s)` or `sponsor(s)` to avoid getting flagged by ad blockers.

**good**

```html
<p id="eventsponsor">
```

**bad**

```html
<p id="event-sponsor">
```

```html
<p id="sponsor">
```

```html
<p id="sponsors">
```

```html
<p id="ad">
```

```html
<p id="ads">
```
