# id-unique

IDs should be unique.

**good**

```html
  <p id="one">Hello</p>
  <p id="two">This is html-lint</p>
```

**bad**

```html
  <p id="one">Hello</p>
  <p id="one">This is html-lint</p>
```
