# src-not-empty

`src` attribute should not be empty.

Tags Supported:
  - audio
  - embed
  - iframe
  - img
  - input
  - source
  - track
  - video

**good**

```html
<img src="logo.png" alt="Brand Name">
```

**bad**

```html
<img src="">
```
